src/Features/Core/Portable/Workspace/BackgroundCompiler.cs

```diff
@@ -54,17 +51,17 @@ public void Dispose()
                 _workspace.DocumentOpened -= OnDocumentOpened;
                 _workspace.WorkspaceChanged -= OnWorkspaceChanged;
 
-                _workspace = null;
+                _workspace = null!;
```

Comment:
I'll admit that I found this to be upsetting. I see why it is there, but wish there was a more elegant option here.

Reply:
Guidance for NRT is to ignore the state of the variable after `Dispose` is called.